manifests/satellite6.pp

```diff
@@ -0,0 +1,19 @@
+# Class: satellite6
+#   Install the SAtellite 6 flavor of the robottelo_slave
+#
+class robottelo_slave::satellite6 {
```

Comment:
Maybe name this builder or something similar? `satellite6` is quite generic, and the entirety of the puppet module is designed for Sat support.